src/common/fs/watch.injectable.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { watch } from "chokidar";
+
+const watchInjectable = getInjectable({
```

Comment:
Nit: this foreign 3rd party abstraction is accessed directly from our domestic code, hard-coupling us to chokidar exactly. We could just as well invent our own abstraction that would be simple, stable and controlled in single place.

This may not seem super meaningful here, but as a policy, not letting foreign abstractions bleed in domestic code can sure be a life-saver, and a good investment on top of that.

Reply:
Discussion handled IRL.